.github/workflows/CI.yml

```diff
@@ -82,9 +82,11 @@ jobs:
       - uses: r-lib/actions/setup-r@473c68190595b311a74f208fba61a8d8c0d4c247 # v2.11.1
         with:
           use-public-rspm: true
-          r-version: "4.3"
+          r-version: "4.4"
       - run: echo "LD_LIBRARY_PATH=$(R RHOME)/lib:$LD_LIBRARY_PATH" >> $GITHUB_ENV
         if: matrix.os == 'ubuntu-latest'
+      - run: julia -e 'cp("/usr/lib/x86_64-linux-gnu/libstdc++.so.6", joinpath(dirname(Sys.BINDIR), "lib", "julia", "libstdc++.so.6"); force = true)' 
+        if: matrix.os == 'ubuntu-latest' && matrix.version == '1.6'
```

Comment:
https://juliainterop.github.io/RCall.jl/dev/installation/#Julia-1.8:-Failure-on-recent-Linux-distributions for the reason for this change.